integration/build.gradle

```diff
@@ -13,7 +13,7 @@ dependencies {
         apacheMath = '3.6.1'
         findbugsAnnotations = "3.0.1"
         guava = "33.0.0-jre"
-        jackson = "2.10.2"
+        jackson = "2.15.2"
```

Comment:
I added this because of a problem I was seeing at the time, but it didn't help… the solution was to update Gradle.